students/rhamersky/Lesson_3/mailroom_part1.py

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env python3
+#  Description: This program will allow...........
+# Developer: Ryan Hamersky
+# Date: 05/10/2018
+# Rev: A - 05/15/2018 updated tup_donor_names so it would fit on one page and changed 'is' to '==' when comparing the
+#      variable intChoice.
+
+# -----Data Section-----
+strName = None     # --> User's name
+intOption = int()  # --> User's option.
+tup_donor_names = (["William Gates, III",1,10000.0],["Mark Zuckerberg",1,20000.0],
+                   ["Jeff Bezos",1,1200.0],["Paul Allen",1,5000.0],
+                   ["Mike Scott",1,25.0])  # --> Tuple that stores all donor's information
+
+
+
+# -----Process Section-----
+#  Methods defined in the menu class are related to general menu checks or options based on user inputs.
+
+class menu(object):
```

Comment:
should not be using classes until OO lessons, these should be simple module level functions

Reply:
Natasha,

Once I learned the basic of class in the Python Fundamental class from UW, I got in a habit of using them. I also like my code organized as it helps me find certain areas of my code faster.

If you would really like me to get rid of the classes and make them functions I can in Lesson 6.

Let me know.

Thanks,

Ryan